src/live_restore/live_restore_fs.c

```diff
@@ -1528,9 +1530,15 @@ __live_restore_setup_lr_fh_file_data(WT_SESSION_IMPL *session, WTI_LIVE_RESTORE_
   bool dest_exist, bool source_exist)
 {
     WT_RET(__live_restore_fs_open_in_destination(lr_fs, session, lr_fh, name, flags, !dest_exist));
-    if (have_stop || __wti_live_restore_migration_complete(session) || !source_exist)
+    if (have_stop || !source_exist)
```

Comment:
This check effectively duplicated the check in `__live_restore_setup_lr_fh_file`. We should treat the stop file status and the source exists as the truth. In fact we could overwrite source_exist with have_stop I think 🤔.